### DISCUSSION TITLE
Use "auto rewind" setting when book audio is interrupted by other apps

### DISCUSSION THREAD

**Discussion by wberrier** [PaulWoitaschek/Voice#1368] [Ideas]
App-Version: 3.3.6.2
Phone Name: RW 1st Gen Moto X
Android Version: 5.1

Use case: listening to audiobook in car with bluetooth while google maps is giving directions

The book will pause while google maps gives directions, but sometimes it pauses in the middle of a word, making it easy to miss what was said once audio continues.

It would be great to be able to apply that same "auto rewind" setting for that case.

**Comment by PaulWoitaschek** [comment]
I can set this to a fixed amount of 1s in a future version. The default auto rewind setting is not suited for this because it might be 20seconds and if there are a few notifications coming after each other, the playback would be set way too far back.

**Comment by wberrier** [comment]
That sounds great.

**Comment by innesst** [comment]
On Android Auto with Google maps navigation, I will miss 1 to 2 seconds every time maps navigation talks. Rather than instantly stopping when navigation talks, the book volume will lower for 1-2 seconds. I then must rewind after ever direction to hear the whole book. I don't know if this is due to Voice, Maps or Android Auto (though I have had this happen on AA in many rental cars). This volume lowering doesn't happen when connected only by Bluetooth. In any event, being  able to have a small rewind when Voice is interrupted would address the issue. 

version 8.2.3
